include/aikido/control/BarrettHandKinematicSimulationPositionCommandExecutor.hpp

```diff
@@ -47,7 +47,9 @@ class BarrettHandKinematicSimulationPositionCommandExecutor
   /// \return Future which becomes available when the execution completes.
   std::future<void> execute(const Eigen::VectorXd& goalPositions) override;
 
-  // Documentation inherited.
+  /// If multiple threads are accessing this function or the skeleton associated
+  /// with this executor, it is necessary to lock the skeleton before
+  /// calling this method.
```

Comment:
Nit: I believe this comment is a note to be _added_ to the original docstring of `PositionCommandExecutor::step()`, but this will just overwrite the original docstring. It would be nice to copy the original docstring and add this node as:
```cpp
  /// \copydoc PositionCommandExecutor::step()
  ///
  /// If multiple threads are accessing this function or the skeleton associated
  /// with this executor, it is necessary to lock the skeleton before
  /// calling this method.
```

Also, it would be great to actually add a docstring to `PositionCommandExecutor::step()`. Currently, it's `// Step once`. If it's enough then make it doxygen comment by prepending one more `/`.